alchemy-web/src/content/docs/guides/cloudflare-astro.mdx

```diff
@@ -195,6 +193,10 @@ console.log({
 await app.finalize();
 ```
 
+:::note
+If you are not using server-side rendering with Astro, set `output: "static"` in the `Astro` resource. This option should match the `output` option in your `astro.config.mjs` file.
```

Comment:
Let's track an issue to infer it if we won't do that as part of this change.

Reply:
Good catch. It is being inferred now, just forgot to remove that note.